lib/test_ids/allocator.rb

```diff
@@ -6,9 +6,11 @@ class Allocator
     def initialize
       @@allocators ||= 0
       @@allocators += 1
```

Comment:
Don't think these vars are required for anything else are they? Let's just get rid of them.